ssl/src/main/java/org/wildfly/security/ssl/SSLExplorer.java

```diff
@@ -54,6 +54,7 @@
 final class SSLExplorer {
 
     private static final MechanismDatabase database = MechanismDatabase.getInstance();
+    private static final MechanismDatabase tls13_database = MechanismDatabase.getTLS13Instance();
```

Comment:
This may make my previous comment redundant about avoiding loading both databases unless both are needed - but maybe there should be no retained static reference here at all as MechanismDatabase already has static references.

Reply:
Removed these variables and used `MechanismDatabase.getInstance()` and `MechanismDatabase.getTLS13Instance()` directly.